src/pages/en/core-concepts/astro-components.md

```diff
@@ -152,6 +152,17 @@ const items = ["Dog", "Cat", "Platypus"];
 </ul>
 ```
 
+:::tip
+You can also set tags dynamically:
+
+```astro
+---
+const El = 'div'
+---
+<El>Hello!</El> <!-- El will be transformed into a div -->
+```
```

Comment:
``````suggestion
```astro "El"
---
// src/pages/index.astro
const El = 'div'
---
<El>Hello!</El> <!-- renders as <div>Hello!</div> -->
```
``````
Making the code formatting changes to show a file name title, and (I think!) highlight the word "El." Committing to the branch so that I can verify in the deploy preview that this does what I think it does!